examples/06_benchmarks/movielens.ipynb

```diff
@@ -76,24 +76,23 @@
      "name": "stdout",
```

Comment:
<p>There is a massive time difference in LigthGCN, are we using the same machine Standard NC6 as before?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1790/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Yes, we are using the same machine Standard NC6 here.